### ISSUE TITLE
Include 3e7e6f3 for 0.6.5.4

### ISSUE THREAD

**Issue by tuxoko** [zfsonlinux/spl#517]
@nedbass 
I see you are preparing for next point release.
Could you include 3e7e6f3 as well. This one is pretty self contained and fix zfsonlinux/zfs#3936, in which a large core count system would easily livelock in slub.
Thanks.

**Comment by behlendorf** [comment]
Good idea, thanks for calling that one out @tuxoko.  It's a small fix which has been in master for quite a while now and definitely addresses a real bug.

**Comment by AndCycle** [comment]
missed 0.6.5.4, gonna put this one by hand again
